### DISCUSSION TITLE
04/065-%E7%AC%AC%E5%85%AD%E5%8D%81%E4%BA%94%E7%AB%A0-%E5%8F%A6%E8%BE%9F%E8%B9%8A%E5%BE%84.html

### DISCUSSION THREAD

**Discussion by giscus** [liuli-moe/to-the-stars#24] [General]
04/065-%E7%AC%AC%E5%85%AD%E5%8D%81%E4%BA%94%E7%AB%A0-%E5%8F%A6%E8%BE%9F%E8%B9%8A%E5%BE%84.html
“在战时，真相是如此珍贵，必须由谎言的警卫加以保护。”
https://tts.liuli.moe/04/065-%E7%AC%AC%E5%85%AD%E5%8D%81%E4%BA%94%E7%AB%A0-%E5%8F%A6%E8%BE%9F%E8%B9%8A%E5%BE%84.html

**Comment by ArgusK17** [comment]
这章节奏简直快到可怕（
不过这才是TTS的醍醐味

**Comment by NCC-114514** [comment]
本章要素：
学姐排面
紫毛捶地哭
女人打架
寡妇文学（伪）
激情神交
外星怪话
（错乱

**Comment by bixudamajiang** [comment]
章鱼的多线程像极了群友聊天（

**Reply by NCC-114514** [reply]
智慧生物的未来（大悲）

**Reply by bixudamajiang** [reply]
你的未来是指赢赢赢和输输输吗？
那确实是大悲了

**Reply by NCC-114514** [reply]
那种事情不要啊！

**Comment by moermeng** [comment]
说实话我很担心这么快的节奏会不会是作者绷不住了的预兆，不过他最近好像还蛮精神的（
第四卷一开始就从多个不同视角推进故事，我原来还以为会把良子的部分往后推两章吊读者胃口，不过既然当了三卷反派的章鱼，哦不，是思裔，终于走上台前，想必接下来的故事也将围绕着良子的章鱼国见闻和晓美焰的神秘目的展开。再加上破坏tcf的神秘组织，本该死去的魔法少女，西蒙娜的身世，非法殖民地的谜团，圆神的黑手（不是，这些大坑可都没填呢
故事的走向相比会更加扑朔迷离，好想看下一章啊啊啊啊啊

**Reply by bixudamajiang** [reply]
作者在现实里找TCF的漏洞呢，慢慢等吧

**Comment by LMOliver** [comment]
> 通迅

typo?

**Reply by rxliuli** [reply]
是的，你想提一个 pr 修复它么？或者吾辈晚点处理一下
